hotel-service/src/main/resources/static/style.css

```diff
@@ -0,0 +1,200 @@
+* {
```

Comment:
Please exclude this file from the commit: it is not related to the current task.